.github/workflows/gorgone.yml

```diff
@@ -133,115 +130,118 @@ jobs:
-            CENTREON_REPO_LINKED_BRANCH="${{ github.head_ref || github.ref_name }}"
-          fi
-
-          echo "linked_branch=$CENTREON_REPO_LINKED_BRANCH" >> $GITHUB_OUTPUT
-        shell: bash
-
-      - name: Checkout sources
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
-
-      - name: Checkout sources
-        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
-        with:
-          repository: centreon/centreon
-          path: centreon
-          ref: ${{ steps.centreon_repo_linked_branch.outputs.linked_branch }}
-          sparse-checkout: |
-            centreon/www/install/createTables.sql
-            centreon/www/install/createTablesCentstorage.sql
-
-      - name: get cached gorgone package
-        uses: actions/cache/restore@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
-        with:
-          path: ./*.${{ matrix.package_extension }}
-          key: ${{ github.sha }}-${{ github.run_id }}-${{ matrix.package_extension }}-${{ matrix.distrib }}
-          fail-on-cache-miss: true
-
-      - name: Install gorgogne from just built package
-        shell: bash
-        run: |
-          if [[ "${{ matrix.package_extension }}" == "deb" ]]; then
-            apt update
-            apt install -y ./centreon-gorgone*${{ matrix.distrib }}*
-          else
-            dnf install -y ./centreon-gorgone*${{ matrix.distrib }}* ./centreon-gorgone-centreon-config*${{ matrix.distrib }}*
-            # in el8 at least, there is a package for the configuration and a package for the actual code.
-            # this is not the case for debian, and for now I don't know why it was made any different between the 2 Os.
-          fi
-
-      - name: Create databases
-        run: |
-          mysql -h mariadb -u root -ppassword -e "CREATE DATABASE \`centreon\`"
-          mysql -h mariadb -u root -ppassword -e "CREATE DATABASE \`centreon-storage\`"
-          mysql -h mariadb -u root -ppassword -e "GRANT ALL PRIVILEGES ON centreon.* TO 'centreon'@'%'"
-          mysql -h mariadb -u root -ppassword -e "GRANT ALL PRIVILEGES ON  \`centreon-storage\`.* TO 'centreon'@'%'"
-          mysql -h mariadb -u root -ppassword 'centreon' < centreon/centreon/www/install/createTables.sql
-          mysql -h mariadb -u root -ppassword 'centreon-storage' < centreon/centreon/www/install/createTablesCentstorage.sql
-
-      - name: Run tests
-        run: robot -v 'DBHOST:mariadb' -v 'DBNAME:centreon' -v 'DBNAME_STORAGE:centreon-storage' -v 'DBUSER:centreon' gorgone/tests
-
-      - name: Upload gorgone and robot debug artifacts
-        if: failure()
-        uses: actions/upload-artifact@0b2256b8c012f0828dc542b3febcab082c67f72b # v4.3.4
-        with:
-          name: gorgone-debug-${{ matrix.distrib }}
-          path: |
-            log.html
-            /var/log/centreon-gorgone
-            /etc/centreon-gorgone
-          retention-days: 1
+  # test-gorgone:
```

Comment:
Please do not disable gorgone tests on develop

Reply:
test are now uncommented, issue is acknowledged and tests will be fixed outside of this pr